src/python/pants/backend/python/goals/setup_py_test.py

```diff
@@ -384,7 +384,7 @@ def test_generate_chroot_entry_points(chroot_rule_runner: RuleRunner) -> None:
                     name='foo', version='1.2.3',
                     entry_points={
                         "console_scripts":{
-                            "foo_qux":"foo.baz.qux",
+                            "foo_qux":"foo.baz.qux:main",
```

Comment:
Good to see that we catch bugs in the test suite :P

Reply:
So, before it was inferring a dep on the `python_library` and now it does it on the `pex_binary`? Cool!

Reply:
Well, there are too many `foo`, `bar`, `baz` and `qux` in this test. But no, I don't think so. It refers to a module (that is not part of any target, so "orphan") but being a console script, it must point at a method, not just a module.

Reply:
> Well, there are too many foo, bar, baz and qux in this test

Yep, agreed. I personally don't like foo and friends. https://www.techyourchance.com/never-use-foo-bar-baz/ Feel free to update in another change if you'd like :)